financial-connections/src/main/java/com/stripe/android/financialconnections/navigation/Destination.kt

```diff
@@ -220,6 +220,36 @@ internal sealed class Destination(
     }
 }
 
+internal val Pane.rendersInBottomSheet: Boolean
```

Comment:
This isn’t ideal. Ultimately, I envision that we’ll have our own `NavDestination` class (or extend `Destination`) and add this as a required property.